libcore/src/pedestrian/Pedestrian.cpp

```diff
@@ -1101,16 +815,17 @@ int Pedestrian::GetColor() const
             double v0 = GetV0Norm();
             if(v0 != 0.0) {
                 double v = GetV().Norm();
-                color    = (int) (v / v0 * 255);
+                color    = static_cast<int>(v / v0 * 255);
             }
             return color;
-        } break;
```

Comment:
you have removed a `break` here. this didn't change the logical part?

Reply:
no, as it is unreachable code. Due to the `return` before the function ends before the `break` can be reached.